static/templates/single_message.handlebars

```diff
@@ -26,7 +26,7 @@
                                     {{/if_and}}
                                 </span>
                             {{else}}
-                                <span class="sender_name auto-select">{{msg/sender_full_name}}</span>
+                                <span class="sender_name text-select">{{msg/sender_full_name}}</span>
```

Comment:
@XavierCooney: I have a little bit of a concern about this. Can you elaborate on what effect changing `auto-select` would have in this particular file? What text is this file making selectable?